index.html

```diff
@@ -45,6 +45,170 @@ <h2 class="secondary-titel">
           <a href="https://www.github.com/youmari" class="github-link"><img src="./images/github.svg" alt="github icon"></a>
         </li>
       </ul>
+      <form action="#recent-works" method="get">
+        <button type="submit" class="go-to-main-btn"><img src="./images/arrow.png" alt="arrow down icon"></button>
+      </form>
+    </section>
+    <section class="my-recent-works" id="recent-works">
+      <h2 class="recent-works-titel">My Recent Works</h2>
+      <hr class="green-line">
+      <div class="projects-container">
+        <div class="project1 projects-cards">
+          <img class="project-img" src="#" alt="project1-image">
+          <div class="project-titel-container">
+            <h4 class="project-titel">My Portfolio</h4>
+          </div>
+          <div class="tags-container">
+            <div class="html-tag tags">html</div>
+            <div class="css-tag tags">css</div>
+          </div>

```

Comment:
- [ ] Good job on these lists 💪 however the list of project technologies should also be inside an `<ul>` tag.